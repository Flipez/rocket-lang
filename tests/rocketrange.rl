foreach i in 0 -> 5 ^ 1
  puts(i)
end

puts("---")

foreach i in 0 => 5 ^ 1
  puts(i)
end

puts("---")

foreach i in 5 -> 0 ^ 1
  puts(i)
end

puts("---")

foreach i in 5 => 0 ^ 1
  puts(i)
end

puts("---")

foreach i in 0 -> 5 ^ 2
  puts(i)
end

puts("---")

foreach i in 0 => 5 ^ 2
  puts(i)
end

puts("---")

foreach i in 0 => 5
  puts(i)
end

puts("---")

foreach i in => 5
  puts(i)
end
