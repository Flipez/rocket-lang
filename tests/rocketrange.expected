0
1
2
3
4
"---"
0
1
2
3
4
5
"---"
5
4
3
2
1
"---"
5
4
3
2
1
0
"---"
0
2
4
"---"
0
2
4
"---"
0
1
2
3
4
5
"---"
0
1
2
3
4
5
